sentinelhub/aws_safe.py

```diff
@@ -272,26 +303,41 @@ def get_safe_struct(self):
                         mask_name = self.get_img_name(mask, resolution)
                     else:
                         mask_name = self.get_qi_name(f"{mask}PRB", resolution.lstrip("R"), MimeType.JP2)
-                    safe[main_folder][AwsConstants.QI_DATA][mask_name] = self.get_qi_url(
-                        f'{mask}_{resolution.lstrip("R")}.jp2'
-                    )
+                    qi_safe[mask_name] = self.get_qi_url(f'{mask}_{resolution.lstrip("R")}.jp2')
 
         if self.is_early_compact_l2a():
-            safe[main_folder][AwsConstants.QI_DATA][self.get_img_name(AwsConstants.PVI)] = self.get_preview_url("L2A")
+            qi_safe[self.get_img_name(AwsConstants.PVI)] = self.get_preview_url("L2A")
 
-        preview_type = (
-            "L2A"
-            if (
-                self.data_collection is DataCollection.SENTINEL2_L2A
-                and (self.baseline >= "02.07" or self.baseline == "00.01")
-            )
-            else "L1C"
+        is_newer_l2a_version = self.data_collection is DataCollection.SENTINEL2_L2A and (
+            self.baseline >= "02.07" or self.baseline == "00.01"
         )
```

Comment:
Does it format it any nicer if you put the parenthesis around the whole expression? Something like
```python
x = (
    exp1
    and exp2
)
```

Reply:
Sadly no - black just automatically reverts back to the previous formatting. I would leave it like it is.

Reply:
sad :( but yeah, useless to fight against an autoformatter